test/fuzz/common_test.go

```diff
@@ -53,7 +53,7 @@ type Actor struct {
 }
 
 // stringer for actor
-func (a Actor) String() string {
+func (an Actor) String() string {
 	return a.name
 }
```

Comment:
Hey this change would result in a `nil` pointer dereference error

Reply:
also it makes no sense lol 

a is the name of the actor, not a partical describing a particular actor

Reply:
actually it wouldn't even compile unless there's an unexported package local variable called `a`